ExposurePlayback.xcodeproj/project.pbxproj

```diff
@@ -942,10 +975,54 @@
 			path = Playable;
 			sourceTree = "<group>";
 		};
+		785B37F421A301D900FB7C7B /* ViewControllers */ = {
+			isa = PBXGroup;
+			children = (
+				785B382021A59CB700FB7C7B /* EnvironmentViewController.swift */,
+				785B37F821A3020C00FB7C7B /* LoginViewController.swift */,
```

Comment:
LoginViewController.swift
AssetListTableViewController.swift
NewPlayerViewController.swift

are not present in the project

Reply:
noted